resources/keywords/dbaas_policy.resource

```diff
@@ -66,5 +76,40 @@ Create Custom DBaaSPolicy
     Input Text    ${txtBxConnNS}    *
     Scroll Element Into View    ${btnCreateEnabled}
     Click Element    ${btnCreateEnabled}
-    ${lnkDBaaSPolicyUpd} =    Replace String    ${lnkDBaaSPolicyName}    <<policyname>>   ${policyName}
-    Wait Until Page Contains Element    ${lnkDBaaSPolicyUpd}    timeout=10s
+
+User Creates DBaaSPolicy For ${namespace} From Invalid Namespace Alert on Admin View As ${DBaaSPersona}
+    User Filters Project ${namespace} On Project DropDown And Navigates To Database Access Page
+    User Navigates To Import Database Provider Account Screen From Database Access Page
+    Wait Until Page Contains Element    ${errCreateDBaaSPolicy}     timeout=20s
+    Click Element    ${errCreateDBaaSPolicy}
+    Create Custom DBaaSPolicy as ${DBaaSPersona}
+
+User Is Able To Import ${databaseProvider} Provider Account For ${namespace}
+    User Filters Project ${namespace} On Project DropDown And Navigates To Database Access Page
+    User Navigates To Import Database Provider Account Screen From Database Access Page
+    Import ${databaseProvider} Provider Account
+
+DBaaSPolicy Creation Forbidden
+    Element Should Be Visible    ${errDBaaSPolicyForbid}
+
+Click If Element Present
```

Comment:
Is this a separate keyword? If it is then it should start with 'User...' right?
If not, then the space line should be removed

Reply:
Yes @ikanias It is a separate keyword, but this is more kind of conditional statement rather than User action and being repetitive on multiple places. So I created a reusable for it.

Reply:
OK I see